tutorials/histfactory/hf001_example.C

```diff
@@ -89,7 +90,103 @@ void hf001_example() {
   // meas.PrintXML( "xmlFromCCode", meas.GetOutputFilePrefix() );
 
   // Now, do the measurement
-  MakeModelAndMeasurementFast( meas );
-
-
+  std::unique_ptr<RooWorkspace> ws{MakeModelAndMeasurementFast(meas)};
+
+  RooStats::ModelConfig *modelConfig = static_cast<RooStats::ModelConfig *>(ws->obj("ModelConfig"));
+
+  // Get probability density function and parameters list from model
+  RooAbsPdf *pdf = modelConfig->GetPdf();
+  RooArgSet globalObservables{*modelConfig->GetGlobalObservables()};
+
+  /*
+  parameters in globalObservables:
+
+  nominalLumi
+  nom_alpha_syst1 -> was set as constant in the beginning
+  nom_alpha_syst2
+  nom_alpha_syst3
+  nom_gamma_stat_channel1_bin_0
+  nom_gamma_stat_channel1_bin_1
+
+  */
+
+  // Perform the fit
+  using namespace RooFit;
+  std::unique_ptr<RooFitResult> result{
+     pdf->fitTo(*ws->data("obsData"), Save(), PrintLevel(-1), GlobalObservables(globalObservables))};
+
+   // Getting list of Parameters of Interest and getting first from them
+   RooRealVar* poi = static_cast<RooRealVar*>(modelConfig->GetParametersOfInterest()->first());
+
+   std::unique_ptr<RooAbsReal> nll{pdf->createNLL(*ws->data("obsData"))};
+   std::unique_ptr<RooAbsReal> profile{nll->createProfile(*poi)};
+
+   // frame for future plot
+   std::unique_ptr<RooPlot> frame{poi->frame()};
+
+   frame->SetTitle("");
+   frame->GetYaxis()->SetTitle("-log likelihood");
+   frame->GetXaxis()->SetTitle(poi->GetTitle());
+
+   TCanvas profileLikelihoodCanvas{"combined", "",800,600};
```

Comment:
```suggestion
   RooPlot *frame = poi->frame();

   frame->SetTitle("");
   frame->GetYaxis()->SetTitle("-log likelihood");
   frame->GetXaxis()->SetTitle(poi->GetTitle());

   TCanvas *profileLikelihoodCanvas = new TCanvas{"combined", "",800,600};
```
I appreciate the effort to avoid memory leaks, but unfortunately the plot needs to leak to it is still visible on the screen even after running the macro. Only like this is will also be rendered on the tutorial page.